src/pages/index.page.tsx

```diff
@@ -13,215 +13,198 @@ import useBreakpoint from '~/client/hooks/useBreakpoint';
-			imageUrl="/images/entrepreneurs.webp"
-			link="/entreprendre"
-			linkLabel="En savoir plus"
-			title="Entreprendre : financements, aides et accompagnement">
-			<p>Retrouvez les conseils, outils et structures d’accompagnement pour vous aider à entreprendre</p>
-		</LinkCard>,
+	const accompagnementCardListContent = [
+		{
+			children: <p>Un parcours personnalisé pour vous aider à définir votre projet et trouver un emploi</p>,
+			imageUrl: '/images/cej.webp',
+			link: '/contrat-engagement-jeune',
+			linkLabel: 'Découvrir le CEJ',
+			title: 'Contrat d’Engagement Jeune (CEJ)',
+		},
+		{
+			children: <p>Une association vous recontacte pour vous proposer le programme de mentorat adapté à vos besoins</p>,
+			imageUrl: '/images/mentorat.webp',
+			link: '/mentorat',
+			linkLabel: 'En savoir plus',
+			title: 'Echanger avec un mentor',
+		},
+		{
+			children: <p>Retrouvez les structures proches de chez vous pouvant vous aider dans vos démarches ou votre
+				parcours</p>,
+			imageUrl: '/images/accompagnement-structure.webp',
+			link: '/accompagnement',
+			linkLabel: 'Découvrir mes aides',
+			title: 'Trouver une structure d’accompagnement',
+		},
+		{
+			children: <p>Retrouvez les conseils, outils et structures d’accompagnement pour vous aider à entreprendre</p>,
+			imageUrl: '/images/entrepreneurs.webp',
+			link: '/entreprendre',
+			linkLabel: 'En savoir plus',
+			title: 'Entreprendre : financements, aides et accompagnement',
+		},
 	];
 
-	const aidesEtOutilsCardList = [
-		<LinkCard
-			className={styles.card}
-			key={1}
-			imageUrl="/images/aides-financières.webp"
-			link="/mes-aides"
-			linkLabel="Découvrir mes aides"
-			title="Simulateur d’aides financières">
-			<p>Avec La Boussole, trouvez les aides auxquelles vous avez droit : logement, santé, mobilité, emploi, culture, etc.</p>
-		</LinkCard>,
-		<LinkCard
-			className={styles.card}
-			key={5}
-			imageUrl="/images/créer-son-cv.webp"
-			link="/creer-mon-cv"
-			linkLabel="En savoir plus"
-			title="Je crée mon CV personnalisé">
-			<p>Mettez en avant vos compétences dans un CV, même si vous pensez ne pas avoir d‘expérience</p>
-		</LinkCard>,
+	const aideEtOutilCardListContent = [
+		{
+			children: <p>Avec La Boussole, trouvez les aides auxquelles vous avez droit : logement, santé, mobilité, emploi,
+				culture, etc.</p>,
+			imageUrl: '/images/aides-financières.webp',
+			link: '/mes-aides',
+			linkLabel: 'Découvrir mes aides',
+			title: 'Simulateur d’aides financières',
+		},
+		{
+			children: <p>Mettez en avant vos compétences dans un CV, même si vous pensez ne pas avoir d‘expérience</p>,
+			imageUrl: '/images/créer-son-cv.webp',
+			link: '/creer-mon-cv',
+			linkLabel: 'En savoir plus',
+			title: 'Je crée mon CV personnalisé',
+		},
 	];
 
+	const getCardList = (cardListContent: cardContent[]) => {
+		return cardListContent.map(({ children, ...rest }, index) => (
+			<LinkCard className={styles.card} key={index} {...rest}>{children}</LinkCard>
+		));
```

Comment:
Tu n'as pas besoin de sortir `children` des props pour le passer : 
```suggestion
		return cardListContent.map((props, index) => (
			<LinkCard className={styles.card} key={index} {...props} />
		));
```